### PR TITLE
feat(admin): add option to copy members from one group to other group…

### PR DESCRIPTION
**Pull request by xflord**
…s in the same vo

* added dialog to copy members
* updated openapi

### CODE REVIEW

**Comment by xflord** [comment]
Don't merge until relevant changes to BE are merged.

File: apps/admin-gui/src/assets/i18n/en.json
Comment (Johaney-s) [inline apps/admin-gui/src/assets/i18n/en.json:1183]:
```suggestion
      "COPY_ALL": "Copy all direct members of this groups",
```

**Review by Johaney-s — COMMENTED** [review]
looks nice, maybe this tooltip could be extended to "... cannot be removed or copied"
Screenshot from 2022-12-12 08-48-14(1).

**Review by Johaney-s — APPROVED** [review]

**Review by mattjoke — APPROVED** [review]

**Review by bodnara — COMMENTED** [review]
Other than these small comments I think the use-case can be a little confusing at first. We could discuss the UX aspects of  the dialog. My general idea is to maybe separate the steps to select members (all, or manual selection) and selection of group. Maybe it would be usefull to allow selection of members in the dialog itself. By selecting the manual option a table would appear ? Because right now if you just open the dialog there is a disable option that say nothing is selected by there is no option to select.

**Pull request merged**
